examples/flutter_gallery/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   string_scanner: 1.0.5
   url_launcher: 5.2.7
   cupertino_icons: 0.1.2
-  video_player: 0.10.3+3
+  video_player: 0.10.5
```

Comment:
You will have to run "flutter update-packages --force-upgrade" and include the changes in the PR when changing a pubspec.yaml.

Reply:
@goderbauer this makes sense! I did it but it felt it was rolling forward "too many" versions of other packages, so I held this. Also there's a new gallery coming, so I'm not sure if I should update and merge this, or not :)

Reply:
Yeah, if you update one dependency you'll have to roll all of them forward.

It's upto you. You can update this PR or close it.

Reply:
@goderbauer ok, I'll get this landed ASAP so I can resolve a couple of tickets.

Reply:
PTAL?